types/srp-bigint/tslint.json

```
@@ -0,0 +1,9 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-redundant-jsdoc-2": false,
+        "no-var-keyword": false,
+        "no-var-requires": false,
+        "only-arrow-functions": false
+    }
```

```suggestion
    "extends": "@definitelytyped/dtslint/dt.json"
```
please consider updating your code, with changes like:
```ts
// imports
import srp = require("srp-bigint");
```
```ts
// var(s)
const params = srp.params["1024"];
const N: bigint = params.N;
const g: bigint = params.g;
```
```ts
// exports
/// <reference types="node" />

export interface Params {
    N_length_bits: number;
    N: bigint;
    g: bigint;
    hash: string;
}

export const params: {
    [bits: string]: Params;
};
```
this would make this module declaration types aligned with modern syntax,

thx!

---

This file can now be removed entirely; it is no longer required. (And also, don't disable lint rules, per above!)